flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -213,6 +218,10 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         if let enableOutOfMemoryTracking = arguments["enableOutOfMemoryTracking"] as? Bool {
             options.enableOutOfMemoryTracking = enableOutOfMemoryTracking
         }
+
+        if let tracesSampleRate = arguments["tracesSampleRate"] as? Double {
+            options.tracesSampleRate = NSNumber(value: tracesSampleRate)
+        }
```

Comment:
This is not needed after https://github.com/getsentry/sentry-dart/pull/777
We can also remove https://github.com/getsentry/sentry-dart/pull/749/files#r815995037 if not needed.